velox/type/Type.h

```diff
@@ -1730,7 +1753,9 @@ template <>
 struct SimpleTypeTrait<Timestamp> : public TypeTraits<TypeKind::TIMESTAMP> {};
 
 template <>
-struct SimpleTypeTrait<Date> : public TypeTraits<TypeKind::DATE> {};
+struct SimpleTypeTrait<Date> : public SimpleTypeTrait<int64_t> {
```

Comment:
Why is it int64_t here ? My intuition was that it should be int32_t.

Reply:
Reverted, this was missed during the change, thanks for pointing it out.